### ISSUE TITLE
automate ingest AWS logs into wazuh

### ISSUE THREAD

**Issue by sonofagl1tch** [sonofagl1tch/AWSDetonationLab#5]
update the cloudformation template to automatically update the wazuh server with everything it needs to ingest aws logs

```
<wodle name="aws-s3">
  <disabled>no</disabled>
  <interval>10m</interval>
  <run_on_start>no</run_on_start>
  <skip_on_error>no</skip_on_error>
  <bucket type="cloudtrail">
    <name>wazuh-cloudtrail</name>
    <access_key>insert_access_key</access_key>
    <secret_key>insert_secret_key</secret_key>
  </bucket>
</wodle>
```

**Comment by rossw7** [comment]
Include the steps in the CF template

ssh wazuh
sudo yum install python-pip -y
sudo pip install boto3
sudo vim /var/ossec/etc/ossec.conf
sudo /var/ossec/bin/ossec-control restart

**Comment by sonofagl1tch** [comment]
issue resolved in pull #16
